### DISCUSSION TITLE
Unable to get Openvoice 2.0 working on windows ( Could not load library cudnn_ops_infer64_8.dll. Error code 126 )

### DISCUSSION THREAD

**Discussion by krylor** [myshell-ai/OpenVoice#284] [Q&A]
I am working on windows 11.
Trying to get Openvoice 2.0 working and facing this error consistently.
OpenVoice version: v2
Could not load library cudnn_ops_infer64_8.dll. Error code 126
Please make sure cudnn_ops_infer64_8.dll is in your library path!

What version of Cuda and Cuddn do I need to install, I am getting a lot of conflicting errors on this.

**Comment by Surya-Git-hub** [comment]
hey @krylor i am having same issue, did you find any solution?

**Comment by Zephiel13** [comment]
I have the same problem. cudnn_ops_infer64_8.dll is in cudnn V8 , it seems that openvoice uses cudnn v8 and cuda 11.x

**Comment by krylor** [comment]
I just used Google collab finally.

**Comment by MeachamusPrime** [comment]
Download CUDNN v8.9 from NVidia's developer website. Install. Reboot. It should then get past that.

**Comment by krylor** [comment]
Thanks, I'll check it out.

**Comment by loscrossos** [comment]
i got this solved for me:
under the hood there is the option to use 2 libaries for audio separation: vad and whisper

i think the undelying library whisper is selected by default but i think broken..

-it does not work in CPU mode 
-and even in CUDA its broken.
 
easy fix: use vad.

For it just

change this line: 
```
target_se, audio_name = se_extractor.get_se(reference_speaker, tone_color_converter, vad=False)
```
and set the vad to true:

```
target_se, audio_name = se_extractor.get_se(reference_speaker, tone_color_converter, vad=True)
```

then it will use vad and should work.

basically anywhere that the function "get_se" is calles you can add a ", vad=True" parameter to fix it.

Notebook1 and 2 use vad=True and work with no problems but notebook3 has vad=False and is broken.
Just change it to True

hope this helps!

**Comment by krylor** [comment]
Thanks for the tip!
